arangod/RestHandler/RestOptionsBaseHandler.h

```diff
@@ -0,0 +1,39 @@
+////////////////////////////////////////////////////////////////////////////////
+/// DISCLAIMER
+///
+/// Copyright 2014-2023 ArangoDB GmbH, Cologne, Germany
```

Comment:
2014-2024 :-)

Reply:
You're right. I guess we should address this separately and "fix" all headers at the same time with a script.